cSpell.json

```diff
@@ -0,0 +1,408 @@
+        "Dockerfiles",
+        "Dockerhub",
+        "Dorrans",
+        "dotnetcore",
+        "Dunkin",
+        "Egozi",
+        "else's",
+        "Esposito",
+        "Excella",
+        "Excella's",
+        "eyeroll",
+        "Feedly",
+        "FYIs",
+        "Gmail",
+        "Gojko",
+        "Greenshot",
+        "GUIDs",
+        "Haack",
+        "hackathon",
+        "Hanselman",
+        "HDDs",
+        "Headshots",
+        "Helpdesk",
+        "Hershko",
+        "HKEY",
+        "Hmmm",
+        "htmlproofer",
+        "httpd",
+        "IdentityServer",
+        "Immo",
+        "improv",
+        "incentivizing",
+        "inci",
+        "inetpub",
+        "ISOs",
+        "Itamar",
+        "ITIL",
+        "Kanban",
+        "Karel",
+        "Killeen",
+        "Killeen's",
+        "konmaripoweb",
+        "Landwerth",
+        "lifecycles",
+        "likert",
+        "Linq",
+        "Liskov",
+        "livestream",
+        "Lucene",
+        "Maximo",
+        "microservice",
+        "microservices",
+        "Minecraft",
+        "mistersql",
+        "mkdir",
+        "moreso",
+        "msbuild",
+        "msys",
+        "MVC's",
+        "MVPs",
+        "Nagios",
+        "Namecheap",
+        "Namecheap's",
+        "Nerdgasm",
+        "nevermind",
+        "Neward",
+        "Ninject",
+        "nlkl",
+        "nlkl's",
+        "Noura",
+        "Novotny",
+        "NTFS",
+        "ntlm",
+        "Nuget",
+        "nunit",
+        "Octokit",
+        "octopress",
+        "octopusdeploy",
+        "Ooof",
+        "oooh",
+        "openssl",
+        "Opserver",
+        "org's",
+        "Orgs",
+        "ORMs",
+        "Osherove",
+        "OTRS",
+        "parenttitle",
+        "parenturl",
+        "PCIE",
+        "Photoshop",
+        "Pingendo",
+        "Pluralsight",
+        "POCOs",
+        "Powershell's",
+        "Proto",
+        "Pssst",
+        "Rakefile",
+        "refactorings",
+        "Repos",
+        "Resharper",
+        "RFPs",
+        "ridk",
+        "Roadmap",
+        "Rsync",
+        "Saad",
+        "SATA",
+        "screencast",
+        "Sectigo",
+        "Serilog",
+        "servername",
+        "signkey",
+        "skillset",
+        "skillsets",
+        "Smtp",
+        "solutioning",
+        "Sommelier's",
+        "SpecFlow",
+        "SSRS",
+        "Statiq",
+        "Storify",
+        "sysdate",
+        "teamcity",
+        "Tekkit",
+        "templating",
+        "testwithoutbuild",
+        "thumbprint",
+        "toolset",
+        "Touchpad",
+        "Townhall",
+        "Trello",
+        "Triaging",
+        "Tweetdeck",
+        "Twilio",
+        "unanet",
+        "vcodec",
+        "verifypeer",
+        "Visio",
+        "visualsvn",
+        "VM's",
+        "vmware",
+        "VPNing",
+        "vsts",
+        "Walkthrough",
+        "WMATA",
+        "WMATA's",
+        "Womp",
+        "xluh",
+        "yada",
```

Comment:
Do we want this random list of words here? (e.g. `yada` or `xluh`) The CSpell documentation suggests putting all the errors CSpell finds in an initial run into the dictionary and then removing those that are not valid. 
<img width="842" alt="image" src="https://github.com/kontent-ai/kontent-ai.github.io/assets/25480503/04be65f4-9b2b-4766-8662-84aeb4a1f457">
Then, supposedly, add new words as you encounter them during development. That approach seems more reasonable to me than having a huge dictionary of often crazy words from the start.

Reply:
At first i copied the words from some article and then added ours.. I will remove the unused ones :D

Reply:
I adjusted the list, now it only contains the words related to our needs